plugins/ipam/host-local/main.go

```diff
@@ -54,8 +56,17 @@ func cmdAdd(args *skel.CmdArgs) error {
 		return err
 	}
 
-	r := &types.Result{
-		IP4: ipConf,
+	var r *types.Result
+
+	switch len(ipConf.IP.IP) {
+	case net.IPv6len:
+		r = &types.Result{
+			IP6: ipConf,
+		}
+	default:
```

Comment:
would be nice to have a specific check for ipv4 and ipv6, and error out with an unknown length in the default case.